src/mon/MonMap.h

```diff
@@ -203,6 +204,10 @@ class MonMap {
   epoch_t get_epoch() const { return epoch; }
   void set_epoch(epoch_t e) { epoch = e; }
 
+  // set and get quorum
+  void set_quorum(const std::set<int>& q) { quorum = q; }
+  const std::set<int>& get_quorum() const { return quorum; }
```

Comment:
So I'm not a big fan of having this in a map that should only change every epoch. We are entering a situation where two MonMaps with the same epoch are not equal. I know we discussed in

https://github.com/ceph/ceph/pull/61126#discussion_r1971829742

about duplication issues but I think of all the compromises on versioning or duplicated metadata, adding ephemeral data to the map is my least favorite choice.

Reply:
@batrick, will sending the quorum list as part of MMonMap without changing the monmap structure be enough?

Reply:
@batrick any comment?

Reply:
@NitzanMordhai: I think that's the @batrick's intention.

Reply:
@rzarzynski thanks ! changed it and hope now it will be enough